groups/bdl/bdlb/bdlb_guidutil.h

```diff
@@ -199,6 +199,20 @@ struct GuidUtil {
         // specification, consisting of 122 randomly generated bits, two
         // 'variant' bits set to '10' and four 'version' bits set to '0100'.
 
+    static void generateNonSecure(unsigned char *result,
+                                  bsl::size_t    numGuids = 1);
+        // Generate a sequence of GUIDs meeting the RFC 4122 version 4
+        // specification, and load the resulting GUIDs into the array referred
+        // to by the specified 'result'.  Optionally specify 'numGuids',
+        // indicating the number of GUIDs to load into the 'result' array.  If
+        // 'numGuids' is not supplied, a default of 1 is used.  An RFC 4122
+        // version 4 GUID consists of 122 randomly generated bits, two
+        // 'variant' bits set to '10' and four 'version' bits set to '0100'.
+        // The behavior is undefined unless 'result' refers to a contiguous
+        // sequence of at least 'numGuids' Guid objects.  Note that this
+        // function uses the PCG random engine to generate high quality, albeit
+        // not cryptographically secure, random numbers for Guids.
```

Comment:
We should explain why someone might chose t use this method:

...random numbers for Guids, which is typically *faster* than generating cryptographically secure guids.